package.json

```
@@ -18,7 +18,9 @@
     "clean": "rimraf ./packages/*/build ./packages/*/*.{d.ts,js,esnext,mjs} .sewing-kit",
     "generate": "plop",
     "generate:package": "plop package && yarn plop docs",
-    "yalc": "yalc"
+    "yalc": "yalc",
+    "publish:local": "yarn build --root packages/$PKG_NAME && npx yalc publish --push packages/$PKG_NAME",
```

Turns out `yalc` is listed in the `devDependencies`, so no need to use `npx` to run it – my bad!

```suggestion
    "publish:local": "yarn build --root packages/$PKG_NAME && yalc publish --push packages/$PKG_NAME",
```